buildSrc/settings.gradle

```diff
@@ -22,6 +22,13 @@ include ":public"
 dependencyResolutionManagement {
     versionCatalogs {
         libs {
+            // For JetBrains Fork only. Keep during merge.
+            // Androidx doesn't support windows development, but we should.
+            // `gradle` folder is symlinked for other modules like (:compose:runtime),
+            // and it works fine for Mac and Linux but not for Windows
+            if (System.getProperty("os.name").startsWith("Windows")) {
```

Comment:
It looks that gradle has more readable way to check os: `org.gradle.nativeplatform.platform.internal.DefaultNativePlatform.currentOperatingSystem`
[isWindows()](https://docs.gradle.org/current/javadoc/org/gradle/nativeplatform/platform/OperatingSystem.html#isWindows--)